statsd/worker.go

```diff
@@ -71,9 +73,15 @@ func (w *worker) processMetric(m metric) error {
 }
 
 func (w *worker) shouldSample(rate float64) bool {
+	// rand.NewSource is not thread safe.
```

Comment:
Nit: for clarity, I suggest:
```suggestion
	// sources created by rand.NewSource() (ie. w.random) are not thread safe.
```